conans/test/util/tools_test.py

```diff
@@ -360,12 +359,12 @@ def __call__(self, command, *args, **kwargs):  # @UnusedVariable
             "CONAN_SYSREQUIRES_SUDO": "True"
         }):
             packages = ["verify_package", "verify_another_package", "verify_yet_another_package"]
-            runner = RunnerMultipleMock(["sudo -A apt-get update"])
-            spt = SystemPackageTool(runner=runner, tool=AptTool())
+            runner = RunnerMultipleMock(["sudo --askpass apt-get update"])
```

Comment:
Check if this is from another PR

Reply:
After merging develop (with the revisions branch), there appeared new tests and modifications related to `--askpass`/`-A`.  I think it is not so important to have those mixed. Also, there is #3969, and eventually we will need to change all of them to `-A`. 

Nevertheless, if you think it is a problem, I can uniformize them all.